packages/corelib/src/playout/timings.ts

```diff
@@ -157,7 +157,10 @@ export function getPartTimingsOrDefaults(
 }
 
 function calculateExpectedDurationWithPreroll(rawDuration: number, timings: PartCalculatedTimings): number {
-	return Math.max(0, rawDuration + timings.toPartDelay - timings.fromPartRemaining)
+	// toPartDelay is removed from fromPartRemaining here, otherwise the values cancel out and the 'from' Part will
+	//  count overtime when prerolling into the 'to' Part.
+	// As a result, the 'to' Part will have a countdown which includes its own preroll.
+	return Math.max(0, rawDuration + timings.toPartDelay - (timings.fromPartRemaining - timings.toPartDelay))
```

Comment:
This change seems odd.. It was already using `timings.toPartDelay`, and is now taking the value twice.
That could be reformatted as `rawDuration + (2 * timings.toPartDelay) - timings.fromPartRemaining`

Reply:
Sorry, could you change it back to your formatting of the line, that made the intent clearer. I was simply trying to show how odd the calculation looks.
But ok, the change sounds reasonable and this value is only used by the UI, so I am happy to go with it